### ISSUE TITLE
Suggestion: disallow equipment with more slots than you have dice from appearing in Inventor Reunion

### ISSUE THREAD

**Issue by ncrecc** [TerryCavanagh/diceydungeons.com#2455]
E.g. prevent Four Handed Sword form appearing and being borderline-unusable if the player doesn't naturally have enough dice to activate it
![image](https://user-images.githubusercontent.com/11216597/128228403-5f3abc00-9fa9-4775-9b7a-a24d552a65ed.png)

**Comment by TerryCavanagh** [comment]
Yeah, this particular issue comes up pretty much exclusively because of Four Handed Sword - which is obviously an amazing item to get late game. But kind of a boring one! So I might just exclude Four Handed Sword from the episode completely.
